### DISCUSSION TITLE
Umbraco 13 - Webhook - Logs - Cleanup policy

### DISCUSSION THREAD

**Discussion by JeffreyPerplex** [umbraco/Umbraco-CMS#15228] [Features and ideas]
I would suggest to implement a clean up policy for all webhook logs. I think this log can get big really soon, especially because both requests and responses are stored.

![afbeelding](https://github.com/umbraco/Umbraco-CMS/assets/12457148/ea356cec-d903-4d1f-b6e5-11330a9196d2)

They way you configure the cleanup is up to you I would say :)

**Comment by AndyButland** [comment]
Will close this as I've realised we already support this feature, configured via the webhook settings: https://docs.umbraco.com/umbraco-cms/reference/webhooks#webhook-settings
